### PR TITLE
Bug 1171685 - Update device ID for uplift.

### PR DESCRIPTION
**Pull request by tamarahills**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8f691854fd15141c812e216fdc22ffbd51d45c76

File: apps/sharedtest/test/unit/telemetry_test.js [hunk 1]
```diff
@@ -122,4 +148,41 @@ suite('Telemetry:', function() {
...
     });
   });
+
+  suite('getDeviceID', function () {
+
+    setup(function() {
+    });
+
+    teardown(function() {
+    });
```
Comment (marshall) [inline apps/sharedtest/test/unit/telemetry_test.js]: nit: looks like the setup/teardown aren't actually being used, probably better to remove them

File: shared/js/telemetry.js [hunk 2]
```diff
@@ -101,6 +102,57 @@
...
+        // has returned an imei.
+        if (result && result.success && (result.serviceCode === 'scImei')) {
+          return result.statusMessage;
+        } else {
+          reject('Unable to get IMEI');
+        }
+      }).then(function (imei) {
+        return TelemetryRequest.createHash(imei);
+      }).then(function (hashed) {
+        console.log('TAMARA:HASHED:' + hashed);
```
Comment (marshall) [inline shared/js/telemetry.js]: nit: leftover debug logging

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0015983fa45483d13778d58a9274acc7b21b1c10

**Pull request closed without merging**